### PR TITLE
Updating oakland with APTP text

### PR DESCRIPTION
**Pull request by cjappl**
Hey all,

I worked with [APTP ](https://www.antipoliceterrorproject.org/) to update the text for the oakland letter. They are an Oakland, black led coalition that fights police terror. They have organized many of the actions in the bay area over the last week.

This message has the sign off of their comms committee. I believe they've been in touch with the main maintainer of defund12 as well.

### CODE REVIEW

File: _emails/oakland.md [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
 
-  My name is \[YOUR NAME\], and I am a resident of Oakland. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. Our city has been at the forefront of much of this action. Accordingly, it has come to my attention that the budget for 2021 is being decided as these protests continue.
+  OPD takes an enormous share of the city’s general fund, and that percentage has risen exponentially for the last two decades, taking away desperately needed resources from essential city programs and services. OPD has overspent its authorized overtime budget by 100% -- tens of millions of dollars -- in recent years, as individual cops bring home annual compensation upwards of $400k. The investment in policing has not made us safer -- OPD remains an embarrassment to the city and a lethal threat to Oakland’s Black and Brown communities, while increased police spending shows no correlation to decreasing crime levels over the past 20 years. With Oakland's current finances in dire jeopardy, it is clear that we must defund the police.
```
Comment (swallan) [inline _emails/oakland.md:21]: It might still be worthwhile to include the `My name is \[YOUR NAME\], and I am a resident of Oakland.`

Comment (cjappl) [reply _emails/oakland.md:21]: cool, give me just a sec and I'll revise!

File: _emails/oakland.md [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
-- askrcd-da@acgov.org
-subject: Oakland Resident for Defunding the OPD
+- atlarge@oaklandca.gov
+subject: Defund OPD.  Invest in Community.
 body: |-
-  To Mayor Schaaf, the Oakland City Council and Oakland and Alameda County Elected Officials:
+  Dear Mayor Schaaf and City Council Members,
 
-  My name is \[YOUR NAME\], and I am a resident of Oakland. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. Our city has been at the forefront of much of this action. Accordingly, it has come to my attention that the budget for 2021 is being decided as these protests continue.
+  My name is \[YOUR NAME\] and I am an Oakland resident writing to urge you to defund OPD.
```
Comment (milesalex) [inline _emails/oakland.md:21]: Backslashes should be removed

Comment (cjappl) [reply _emails/oakland.md]: done! Thank you :)

File: _emails/oakland.md [hunk 3]
```diff
@@ -1,41 +1,40 @@
 ---
 title: Oakland, CA
-permalink: "/oakland"
+permalink: "/defundopd"
```
Comment (serifluous) [inline _emails/oakland.md:3]: Any concerns that changing the permalink might affect those who are already sharing the /oakland URL?

Comment (swallan) [reply _emails/oakland.md:3]: It may be better to stick to the style of other permalinks as well

Comment (teddywilson) [reply _emails/oakland.md:3]: Yeah. Any links that are live cannot be changed at the moment. I will create an issue for this. We should work on a solution for redirects if we want to modify existing links.

Comment (cjappl) [reply _emails/oakland.md]: Good thought! Changed it back :)

**Pull request merged**